execution/gethexec/caff_node.go

```diff
@@ -136,133 +111,45 @@ func (n *CaffNode) createBlock() (returnValue bool) {
 
 	if err != nil {
 		log.Error("Failed to produce block", "err", err)
-		// if we fail to produce a block, we should remove this message from the queue
-		// and set skippedBlockPos to the current messageWithMetadataPos
-		n.skippedBlockPos = &messageWithMetadataPos
-		n.messagesWithMetadata = n.messagesWithMetadata[1:]
-		n.messagesWithMetadataPos = n.messagesWithMetadataPos[1:]
 		return false
 	}
 
 	// If block is nil or receipts is empty, return false
-	if len(receipts) == 0 || block == nil {
+	if block == nil {
 		log.Error("Failed to produce block, no receipts or block")
-		// if we fail to produce a block, we should remove this message from the queue
-		// and set skippedBlockPos to the current messageWithMetadataPos
-		n.skippedBlockPos = &messageWithMetadataPos
-		n.messagesWithMetadata = n.messagesWithMetadata[1:]
-		n.messagesWithMetadataPos = n.messagesWithMetadataPos[1:]
```

Comment:
I think it is a right move to retry producing blocks

Reply:
We are not retrying producing blocks? Should we do it?

Reply:
Resetting the values now: https://github.com/EspressoSystems/nitro-espresso-integration/pull/479/commits/de6b5cdd2f18369b7d59ae36c87307151cf6bc67

Reply:
https://github.com/EspressoSystems/nitro-espresso-integration/pull/479/commits/de6b5cdd2f18369b7d59ae36c87307151cf6bc67